rust/protocol/src/storage/traits.rs

```diff
@@ -78,6 +102,7 @@ pub trait SignedPreKeyStore {
     ) -> Result<()>;
 }
 
+/// Interface for storing a session associated with the chat with a particular separate user.
```

Comment:
I'd be careful about "user" vs. "device" here (or "client instance" if you prefer). "Users" are pretty much not interesting in libsignal-protocol; everything is device-to-device.

Reply:
The reason the terminology was incorrect was because I was confused about what was being stored! Thanks, will reflect this.

Reply:
> "Users" are pretty much not interesting in libsignal-protocol; everything is device-to-device.

This sounds great!